.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Copilot Chat Issues
+    url: https://github.com/microsoft/vscode/issues/new?labels=chat
+    about: Please file issues related to Copilot Chat in the VS Code repository.
```

Comment:
Issues are disabled, wouldn't we need to enable them to get this?

Reply:
we would enable the issues, and `blank_issues_enabled` will ensure they get directed to vscode. Will enable it after this PR is merged.